zh-cn/index.md

```diff
@@ -0,0 +1,206 @@
+Unpacking the package...
+Installing the plugin...
+Plugin installed successfully.
+Activating 'user-switching'...
+Plugin 'user-switching' activated.
+Success: Installed 1 of 1 plugins.
+```
+
+WP-CLI 还包含许多您无法在 WordPress 后台管理中执行的操作命令。例如，`wp transient delete --all`（[说明文档](https://developer.wordpress.org/cli/commands/transient/delete/)）可以删除一个或所有的 Transients ：
+
+```bash
+$ wp transient delete --all
+Success: 34 transients deleted from the database.
+```
+
+有关如何使用 WP-CLI 的更多内容请阅读《[Quick Start](https://make.wordpress.org/cli/handbook/quick-start/)》。您也可以在 [Shell Friends](https://make.wordpress.org/cli/handbook/shell-friends/) 了解实用的命令行工具。
+
+如果您已经了解这些基础知识，可以到 [WP-CLI Commands](https://developer.wordpress.org/cli/commands/) 了解更多有关主题和插件管理、数据导入和导出、数据库操作的内容。
+
+## 安装说明
+
+下载 Phar 文件是我们推荐的安装方法，如果需要[其他安装方法](https://make.wordpress.org/cli/handbook/installing/)（[Composer](https://make.wordpress.org/cli/handbook/installing/#installing-via-composer), [Homebrew](https://make.wordpress.org/cli/handbook/installing/#installing-via-homebrew), [Docker](https://make.wordpress.org/cli/handbook/installing/#installing-via-docker)），请参阅相关文档。
+
+在安装 WP-CLI 之前，请确保您的操作环境满足最低要求：
+
+- UNIX 环境（OS X，Linux，FreeBSD，Cygwin），某些功能在 Windows 中将受到限制。
+- PHP 5.4 或更高版本。
+- WordPress 3.7 或更高版本，较旧的版本在功能上可能会有所减少。
+
+再次检查操作环境，使用  `wget` 或 `curl` 下载 [wp-cli.phar](https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar)：
+
+```bash
+curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+```
+
+接下来，检查 Phar 文件确保其正常运行：
+
+```bash
+php wp-cli.phar --info
+```
+
+要使用 `wp` 执行 WP-CLI 命令，必须有执行权限并且 `PATH` 已在环境变量中注册，例如：
+
+```bash
+chmod +x wp-cli.phar
+sudo mv wp-cli.phar /usr/local/bin/wp
+```
+
+如果 WP-CLI 安装成功，使用 `wp --info` 命令应该看到以下输出信息：
+
+```bash
+$ wp --info
+OS:	Darwin 16.7.0 Darwin Kernel Version 16.7.0: Thu Jan 11 22:59:40 PST 2018; root:xnu-3789.73.8~1/RELEASE_X86_64 x86_64
+Shell:	/bin/zsh
+PHP binary:    /usr/local/bin/php
+PHP version:    7.0.22
+php.ini used:   /etc/local/etc/php/7.0/php.ini
+WP-CLI root dir:        /home/wp-cli/.wp-cli/vendor/wp-cli/wp-cli
+WP-CLI vendor dir:	    /home/wp-cli/.wp-cli/vendor
+WP-CLI packages dir:    /home/wp-cli/.wp-cli/packages/
+WP-CLI global config:   /home/wp-cli/.wp-cli/config.yml
+WP-CLI project config:
+WP-CLI version: 2.4.0
+```
+
+### 升级 WP-CLI
+
+如果需要更新 WP-CLI ，可以运行 `wp cli update`（[说明文档](https://developer.wordpress.org/cli/commands/cli/update/)）或者重复上述安装方法。
+
+如果 WP-CLI 的所有者是 root 或其他系统管理员，则需要执行 `sudo wp cli update` 操作。
+
+WordPress 的发展相当迅速，Nightly Builds 版本可以在开发环境中使用，并且始终包含最新和最出色的 WP-CLI 功能，运行 `wp cli update --nightly` 即可使用。
+
+### Tab 命令行补全
+
+WP-CLI 带有用于 Bash 和 ZSH 的命令行补全脚本。下载 [wp-completion.bash](https://raw.githubusercontent.com/wp-cli/wp-cli/v2.4.0/utils/wp-completion.bash) 并 `~/.bash_profile` 加载它即可。例如：
+
+```bash
+source /FULL/PATH/TO/wp-completion.bash
+```
+
+不要忘了之后运行 `source〜/ .bash_profile` 。
```

Comment:
change `source〜/ .bash_profile` to `source 〜/ .bash_profile`

Reply:
@wojsmol Thank you for your reminder, but fortunately this is not the rm command 😔